### PR TITLE
Feature basic project setup

### PR DESCRIPTION
**Pull request by marcinel123**
Hi Adam,

Here are the changes I made in this PR:
- installed most of required dependencies
- added complete theme settings for styled components together with google fonts
- eslint has been configured
- added basic components structure

Please have a look.
Regards,
Martin

### CODE REVIEW

File: src/components/InputandFilterPanel/InputandFilterPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const InputandFilterPanel = () => (
+	<section>
+		<input placeholder="search for a conutry" />
```
Comment (AdamKniec) [inline src/components/InputandFilterPanel/InputandFilterPanel.tsx]: Input does not exist without a form, even if you just create some initial commit

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export const Header = () => (
+	<>
+		<h2>Where in the world?</h2>
+		<button type="button">Dark Mode</button>
```
Comment (AdamKniec) [inline src/components/Header/Header.tsx:4]: Button should have a label

File: src/theme.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+export const theme = {
```
Comment (AdamKniec) [inline src/theme.tsx:1]: nice 👍

File: src/components/InputandFilterPanel/InputandFilterPanel.tsx (hunk 1 above)
Comment (marcinel123) [reply src/components/InputandFilterPanel/InputandFilterPanel.tsx]: solved.

File: src/components/Header/Header.tsx (hunk 2 above)
Comment (marcinel123) [reply src/components/Header/Header.tsx:4]: But we have never used label for button. Even in previous projects. I checked in w3schools for HTML rules and label is not required.

**Pull request merged**